### ISSUE TITLE
.lintr config file not working

### ISSUE THREAD

**Issue by karlvurdst** [jimhester/lintr#880]
I am using vsc with R, and would like to tweak the lintr, specifically, I want the lintr to stop asking me to change single quotes to double qotes.

I found out that I am supposed to use the .lintr config file. But it doesn't seem that vsc is using this file at all? It currently looks like this:

```
linters: with_defaults(
  line_length_linter(120),
  single_quotes_linter = NULL
)
```

But in my vsc, the line length is 80, not 120 (which I guess it is getting from my settings, and this is fine). But it still warns me that I am using single quotes, after I added `single_quotes_linter = NULL` to the file. I have restarted VSV.

**Comment by renkun-ken** [comment]
Your `.lintr` should be put to `~/.lintr` or in workspace folder and should be like the following:

```r
linters: with_defaults(
  line_length_linter(120), 
  single_quote_linter = NULL)
```

Start R from the workspace folder and run linting manually to check if it works:

```r
lintr::lint("xxx")
```
